service/src/test/java/org/cbioportal/service/impl/ClinicalDataServiceImplTest.java

```diff
@@ -255,9 +258,25 @@ public void fetchClinicalDataCounts() throws Exception {
         clinicalDataCount5.setCount(3);
         clinicalDataCounts.add(clinicalDataCount5);
 
+        
+        ClinicalAttribute clinicalAttribute1 =new ClinicalAttribute();
+        clinicalAttribute1.setAttrId(CLINICAL_ATTRIBUTE_ID_1);
+        clinicalAttribute1.setPatientAttribute(true);
+        ClinicalAttribute clinicalAttribute2 =new ClinicalAttribute();
+        clinicalAttribute2.setAttrId(CLINICAL_ATTRIBUTE_ID_2);
+        clinicalAttribute2.setPatientAttribute(true);
+        ClinicalAttribute clinicalAttribute3 =new ClinicalAttribute();
+        clinicalAttribute3.setAttrId(CLINICAL_ATTRIBUTE_ID_3);
+        clinicalAttribute3.setPatientAttribute(true);
+        
+        Mockito.when(clinicalAttributeService.getClinicalAttributesByStudyIdsAndAttributeIds(
+                Arrays.asList(STUDY_ID, STUDY_ID, STUDY_ID), Arrays.asList(CLINICAL_ATTRIBUTE_ID_1, 
+                        CLINICAL_ATTRIBUTE_ID_2, CLINICAL_ATTRIBUTE_ID_3), "SUMMARY"))
+        .thenReturn(Arrays.asList(clinicalAttribute1,clinicalAttribute2, clinicalAttribute3));
+        
         Mockito.when(clinicalDataRepository.fetchClinicalDataCounts(Arrays.asList(STUDY_ID, STUDY_ID, STUDY_ID), 
```

Comment:
Do we have any test cover the fetchClinicalDataCounts when there are conflicts clinical attributes?

Reply:
no, i'll have to add that